package.yaml

```diff
@@ -11,6 +11,7 @@ scripts:
   build:tokens: pnpm --filter @equinor/eds-tokens run build
   build:icons: pnpm --filter @equinor/eds-icons run build
   storybook: pnpm --filter ./libraries/core-react run storybook
+  graphics: pnpm --filter ./graphics/woman run dev
```

Comment:
Looks like a missed path change after moving to `assets`

Reply:
Right, missed that – thanks 😊

Reply:
```suggestion
  graphics: pnpm --filter ./assets/woman run dev
```

Rename graphics → assets